src/components/SideNavMobile.tsx

```diff
@@ -5,15 +5,20 @@ import { motion, AnimatePresence } from "framer-motion"
 import Icon from "./Icon"
 import Link from "./Link"
 import Translation from "./Translation"
-import { supportedLanguages } from "../utils/languages"
+import { Lang, supportedLanguages } from "../utils/languages"
 import { dropdownIconContainerVariant } from "./SharedStyledComponents"
+import { IPropsNavLink as INavLinkProps } from "./SideNav"
 
 import docLinks from "../data/developer-docs-links.yaml"
+import { DeveloperDocsLink } from "../types"
+import { TranslationKey } from "../utils/translations"
 
 // Traverse all links to find page id
-const getPageTitleId = (to, links) => {
-  let linksArr = Array.isArray(links) ? links : [links]
```

Comment:
Here I removed this line because according to the type DeveloperDocsLink, items is always an array so `Array.isArray(links)` will always return true.